src/utils/Util.js

```diff
@@ -8,6 +8,7 @@ import remote from 'remote';
 var app = remote.require('app');
 
 module.exports = {
+  folders: [],
```

Comment:
If this is an array of mounted folders within a VM, shouldn't that be a VirtualBoxUtil.js variable instead? `this.folders = elements;` ?